tests/61push/08_rejected_pushers.pl

```diff
@@ -74,45 +72,50 @@ sub wait_for_push
          create_pusher( $alice, "sytest", "key_2", "$url/2" )
             ->SyTest::pass_on_done( "Alice's pusher 2 created" );
       })->then( sub {
-         do_request_json_for( $alice,
-              method  => "GET",
-              uri     => "/r0/pushers",
-         )->then( sub {
-            my ( $body ) = @_;
-
-            assert_json_keys( $body, qw( pushers ) );
-            @{ $body->{pushers} } == 2 or die "Expected two pushers";
-
-            Future->done(1);
-         });
+         retry_until_success {
```

Comment:
why is this needed? Don't the update and the GET both hit the master?

Reply:
Currently, yes, but there's little reason for that so I figured I may as well change it to not assume that while I'm there anyway.